subsys/spm/spm.c

```diff
@@ -381,6 +457,13 @@ static void spm_config_peripherals(void)
 		NRF_SPU->GPIOPORT[0].PERM = 0;
 	}
 
+#ifdef CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP
```

Comment:
Remember that board names will differ for users, are we using the same style board configs elsewhere in this file? Otherwise a check for the core  (nrf53) would be preferrable.

Reply:
That makes sense. We're not using a similar check in this file (as it's previously only been used with the nRF9160). Will change it.